_includes/common-head.html

```diff
@@ -0,0 +1,8 @@
+<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+<meta http-equiv="content-type" content="text/html; charset=UTF-8">
+<meta charset="UTF-8">
+<title>{{page.title}}</title>
```

Comment:
Try to be consistent, always include spaces inside `{{ page.title }}`, as you do in other places of this PR (or as we always do in Django)

Reply:
👍